src/Build/Graph/ProjectInterpretation.cs

```diff
@@ -239,7 +244,6 @@ ProjectItemInstance projectReference
                         $"{setConfigurationString};{setPlatformString};{setTargetFrameworkString}").ToImmutableDictionary();
                 }
             }
-
```

Comment:
nit: keep the newline

Reply:
reverted all changes in this file as it is no longer where the negotiation takes place